src/main/java/com/novi/springboottechiteasy/models/Television.java

```diff
@@ -0,0 +1,221 @@
+package com.novi.springboottechiteasy.models;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import jakarta.persistence.*;
+
+import java.util.Date;
+import java.util.List;
+
+@Entity
+public class Television {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String type;
+    private String brand;
+    private String name;
+
+    private Double price;
+    @OneToMany(mappedBy = "television")
+    private List<ScreenSize> availableSizes;
+    private Double refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+    private Integer originalStock;
+    @Temporal(TemporalType.DATE)
+    private Date originalStockDate;
+    private Integer sold;
+    @Temporal(TemporalType.DATE)
+    @OneToMany(mappedBy = "television", cascade = CascadeType.ALL)
+    @JsonManagedReference
```

Comment:
Je zou in je entities geen JSON annotaties nodig moeten hebben, omdat je van een entity in principe nooit JSON zult maken.